src/panels/my/ha-panel-my.ts

```diff
@@ -129,17 +132,35 @@ export const getMyRedirects = (hasSupervisor: boolean): Redirects => ({
     redirect: "/config/users",
   },
   general: {
-    redirect: "/config/core",
-  },
-  server_controls: {
-    redirect: "/developer-tools/yaml",
+    redirect: "/config/general",
   },
   logs: {
     redirect: "/config/logs",
   },
   info: {
     redirect: "/config/info",
   },
+  system_health: {
+    redirect: "/config/system_health",
+  },
+  hardware: {
+    redirect: "/config/hardware",
+  },
+  storage: {
+    redirect: "/config/storage",
+  },
+  network: {
+    redirect: "/config/network",
+  },
+  analytics: {
+    redirect: "/config/analytics",
+  },
+  updates: {
+    redirect: "/config/updates",
+  },
+  system_dashboard: {
+    redirect: "/config/system",
+  },
```

Comment:
don't forget to add them to https://github.com/home-assistant/my.home-assistant.io/blob/main/redirect.json

Reply:
And we should update https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/analytics.markdown to point at the new analytics My link.

Reply:
<https://github.com/home-assistant/my.home-assistant.io/blob/main/redirect.json> we cant yet as it does not exist

Reply:
Right, but we should get a draft PR ready.

Reply:
https://github.com/home-assistant/home-assistant.io/pull/22553

Reply:
https://github.com/home-assistant/my.home-assistant.io/pull/214